app/scripts/modules/core/delivery/executionGroup/execution/ExecutionMarker.tsx

```diff
@@ -1,16 +1,21 @@
 import * as React from 'react';
 import * as ReactGA from 'react-ga';
 
-import { IExecutionStageSummary } from 'core/domain/IExecutionStage';
+import { IExecution, IExecutionStageSummary } from 'core/domain';
 import { OrchestratedItemRunningTime } from './OrchestratedItemRunningTime';
-import { Tooltip } from 'core/presentation/Tooltip';
 import { duration } from 'core/utils/timeFormatters';
 
```

Comment:
whitespace between imports. also mixed `import { foo }` and `import {foo}`. I think I'm the only one left who prefers spaced imports for readability, I'm happy to convert to `import {foo}` since it's how all your inferior IDEs prefer it.